src/scalar.jl

```diff
@@ -218,6 +356,52 @@ const asℝ = as(Real, -∞, ∞)
 
 const as_real = asℝ
 
+# Fallback method: print all transforms in order
+function Base.show(io::IO, ct::CompositeScalarTransform)
+    str = string(ct.transforms[1])
+    for ti in ct.transforms[begin+1:end]
+        str *= " ∘ "*string(ti)
+    end
+    print(io, str)
+end
+
+# If equivalent to asℝ₊, print as such. Two ways to achieve this
+function Base.show(io::IO, ct::CompositeScalarTransform{Tuple{TVShift{T}, TVExp}}) where T
+    if ct[1].shift == 0
+        print(io, "asℝ₊")
+    else
+        print(io, "as(Real, ", ct[1].shift, ", ∞)")
+    end
```

Comment:
IMO that's misleading since with this PR you'd get a transform with a different type with `asℝ₊` - this composite transform might behave similar to `asℝ₊` but it is **not** `asℝ₊`.
```suggestion
    print(io, "as(Real, ", ct.transforms[1].shift, ", ∞)")
```